new_platforms/nuget/build/native/openenclave.targets

```diff
@@ -0,0 +1,126 @@
+            </PreprocessorDefinitions>
+
+            <!-- ARM TrustZone Simulation --> 
+            <PreprocessorDefinitions Condition="'$(Configuration)' == 'ARMTZ-Simulation-Debug'">
+                OE_SIMULATE_OPTEE;%(PreprocessorDefinitions)
+            </PreprocessorDefinitions>
+        </ClCompile>
+    </ItemDefinitionGroup>
+
+    <!-- Linking Rules for Enclaves -->
+    <ItemDefinitionGroup Condition="'$(OEType)' == 'Enclave'">
+        <Link>
+            <!-- All, except ARM TrustZone Simulation -->
+            <IgnoreAllDefaultLibraries Condition="'$(Configuration)' != 'ARMTZ-Simulation-Debug'">
+                true
+            </IgnoreAllDefaultLibraries>
+
+            <!-- x86-SGX-Debug -->
+            <AdditionalDependencies Condition="'$(Configuration)' == 'SGX-Debug' And ('$(Platform)' == 'Win32' Or '$(Platform)' == 'x86')">
+                $(SgxHwX86Dbg)\oeenclave.lib;$(SgxHwX86Dbg)\oesocket_enc.lib;$(SgxHwX86Dbg)\oestdio_enc.lib;%(AdditionalDependencies)
+            </AdditionalDependencies>
+
+            <!-- x64-SGX-Debug -->
+            <AdditionalDependencies Condition="'$(Configuration)' == 'SGX-Debug' And '$(Platform)' == 'x64'">
+                $(SgxHwX64Dbg)\oeenclave.lib;$(SgxHwX64Dbg)\oesocket_enc.lib;$(SgxHwX64Dbg)\oestdio_enc.lib;%(AdditionalDependencies)
+            </AdditionalDependencies>
+
+            <!-- x86-SGX-Simulation-Debug -->
+            <AdditionalDependencies Condition="'$(Configuration)' == 'SGX-Simulation-Debug' And ('$(Platform)' == 'Win32' Or '$(Platform)' == 'x86')">
+                $(SgxSimX86Dbg)\oeenclave.lib;$(SgxSimX86Dbg)\oesocket_enc.lib;$(SgxSimX86Dbg)\oestdio_enc.lib;%(AdditionalDependencies)
+            </AdditionalDependencies>
+
+            <!-- x64-SGX-Simulation-Debug -->
+            <AdditionalDependencies Condition="'$(Configuration)' == 'SGX-Simulation-Debug' And '$(Platform)' == 'x64'">
+                $(SgxSimX64Dbg)\oeenclave.lib;$(SgxSimX64Dbg)\oesocket_enc.lib;$(SgxSimX64Dbg)\oestdio_enc.lib;%(AdditionalDependencies)
+            </AdditionalDependencies>
+
+            <!-- x86-ARMTZ-Simulation-Debug -->
+            <AdditionalDependencies Condition="'$(Configuration)' == 'ARMTZ-Simulation-Debug' And ('$(Platform)' == 'Win32' Or '$(Platform)' == 'x86')">
+                $(TzSimX86Dbg)\oeenclave.lib;$(TzSimX86Dbg)\oesocket_enc.lib;$(TzSimX86Dbg)\oestdio_enc.lib;$(TzSimX86Dbg)\oeenclave_opteesim.lib;%(AdditionalDependencies)
+            </AdditionalDependencies>
+        </Link>
+    </ItemDefinitionGroup>
+
+    <!-- Linking Rules for Host Applications -->
+    <ItemDefinitionGroup Condition="'$(OEType)' == 'Host'">
+        <Link>
+            <!-- x86-SGX-Debug -->
+            <AdditionalDependencies Condition="'$(Configuration)' == 'SGX-Debug' And ('$(Platform)' == 'Win32' Or '$(Platform)' == 'x86')">
+                $(SgxHwX86Dbg)\oehost.lib;$(SgxHwX86Dbg)\oesocket_host.lib;$(SgxHwX86Dbg)\oestdio_host.lib;%(AdditionalDependencies)
```

Comment:
This seems to be missing dependencies that those three libs require, e.g. something like:
ws2_32.lib;kernel32.lib;user32.lib;gdi32.lib;winspool.lib;comd
lg32.lib;advapi32.lib;shell32.lib;ole32.lib;oleaut32.lib;uuid.lib;odbc32.lib;odbccp32.lib;sgx_urts.lib;sgx_uae_service.lib

Reply:
I agree with that the `sgx_*.lib` libraries are missing. As for the others, we can't depend on Windows DLL's inside enclaves.

Reply:
Fixed.

I added `onecore.lib` so that we can run in Windows containers, leaving `%(AdditionalDependencies)` at the end if the user configures that beyond defaults. The linker should use exports from `onecore.lib` first, then fall back on the rest for any missing imports.